common/utils_win.cc

```diff
@@ -48,7 +48,13 @@ std::string BuildPipeName(const char* prefix,
                           const std::string& base,
                           bool user_specific) {
   std::string pipename = prefix;
-  pipename += "ProtectedPrefix\\Administrators\\";
+
+  // If the agent is not user-specific, the assumption is that it runs with
+  // administrator provileges.  Create the pipe in a location only available
+  // to administrators.
+  if (!user_specific)
+    pipename += "ProtectedPrefix\\Administrators\\";
```

Comment:
In the original code, when agent doesn't have admin privilege, yet we add the pipe name with "ProtectedPrefix\\Admin", would agent fail to connect with pipe?

Reply:
That's right.  When running a user specific agent, it's very likely the agent won't have admin access.  And we always assume chrome does not have admin access.  Both the agent and chrome need to use the same value of `user_specific` when they create the `Agent::Config` and `Client::Config` objects, respectively, in order to connect with each other.

Reply:
Thanks for the explanation!